scripts/typedoc/typedoc.ts

```diff
@@ -208,6 +245,25 @@ async function components() {
   });
 }
 
+interface FrontMatter {
+  gid: string;
+  url: string;
+  title: string;
+  description?: string;
+  hidden: boolean;
+}
+
+function renderYamlFrontMatter(frontMatter: FrontMatter) {
+  let matter = '---\n';
+
+  (Object.keys(frontMatter) as (keyof FrontMatter)[]).forEach((key) => {
```

Comment:
I probably don't want to know why Typescript won't automatically type the result of `Object.keys(object)` as `keyof` the type of the object, which it knows